htlcswitch/link.go

```diff
@@ -597,8 +602,13 @@ func (l *channelLink) Start() error {
 
 	l.updateFeeTimer = time.NewTimer(l.randomFeeUpdateTimeout())
 
+	// Create a background context that can be canceled when the link is
+	// stopped
+	ctx, cancel := context.WithCancel(context.Background())
+	l.cancel = cancel
+
 	l.cg.WgAdd(1)
-	go l.htlcManager(context.TODO())
+	go l.htlcManager(ctx)
```

Comment:
What does this do then?
https://github.com/lightningnetwork/lnd/blob/5fb0f4317227ac5f57307be36bd7cf60dba63d20/htlcswitch/link.go#L1550